### ISSUE TITLE
Mise à jour de WCAG 2.1 et impact sur les critères 8.1 et 8.2

### ISSUE THREAD

**Issue by AlainVagner** [DISIC/accessibilite.numerique.gouv.fr#176]
Les WCAG 2.1 ont été [mis à jour le 21 septembre 2023](https://www.w3.org/WAI/news/2023-09-21/wcag21updated/). La principale modification est l'ajout d'une note dans le critère de succès 4.1.1 qui dit la chose suivante : 
"This Success Criterion should be considered as always satisfied for any content using HTML or XML."

La norme européenne EN 301 549 fait directement référence dans son [critère 9.4.1.1](https://www.etsi.org/deliver/etsi_en/301500_301599/301549/02.01.02_60/en_301549v020102p.pdf#page=45) au critère de succès WCAG 4.1.1.

[Dans cette discussion sur le gitlab de la norme européenne](https://labs.etsi.org/rep/HF/en301549/-/issues/52#note_4068), l'approche semble être de suivre cette mise à jour de WCAG 2.1 et de considérer dorénavant le critère de succès WCAG 4.1.1 comme toujours conforme.

Au niveau du RGAA, les critères [8.1](https://accessibilite.numerique.gouv.fr/methode/criteres-et-tests/#8.1) et [8.2](https://accessibilite.numerique.gouv.fr/methode/criteres-et-tests/#8.1) fournissent des tests explicites pour ce critère de succès WCAG 4.1.1. Il serait dommage de continuer à réaliser ces tests au vu de cette évolution normative.

**Proposition :** Rajouter une note dans les critères 8.1 et 8.2 précisant que ces critères doivent dorénavant toujours être considérés comme conformes.

**Comment by access42** [comment]
Bonjour, nous partageons l'avis d'Alain Vagner et pensons qu'il est nécessaire que le RGAA s'aligne sur les modifications apportées sur WCAG 2.1 et la norme européenne.
 
Vous trouverez notre position détaillée dans l'article [« WCAG 2.2 : suppression du critère 4.1.1 Analyse syntaxique »](https://access42.net/wcag-2-2-suppression-critere-4-1-1/).

**Comment by AlainVagner** [comment]
Pour information, le critère 9.4.1.1 est dorénavant marqué comme "Void" dans la version 4.1.1 de la norme européenne EN 301 549, qui entrera en vigueur en fin d'année dans toute l'Europe : https://www.etsi.org/deliver/etsi_en/301500_301599/301549/04.01.01_60/en_301549v040101p.pdf#page=68
Les critères 8.1 et 8.2 pourraient donc être simplement supprimés dans la prochaine version du RGAA.
